src/views/CampusSafety/views/LostAndFound/index.tsx

```diff
@@ -28,6 +128,194 @@ const LostAndFound = () => {
+              textTransform: 'uppercase',
+            }}
+            onClick={() => {
+              console.log('Report a Missing Item');
+            }}
+          >
+            Report a Missing Item
+          </Button>
+        </Grid>
+      </Grid>
+
+      {/* My Active Missing Item Reports Section */}
+      <Grid container justifyContent="center" spacing={3} marginTop={3}>
+        <Grid item xs={12} sm={10}>
+          <Card className={styles.card}>
+            <CardContent>
+              <Typography variant="h5" align="center">
+                My Active <span style={{ color: theme.palette.warning.main }}>Missing</span> Item
+                Reports
+              </Typography>
+
+              {/* Table Header */}
+              <Grid
+                container
+                spacing={2}
+                sx={{
+                  marginTop: '10px',
+                  backgroundColor: theme.palette.primary.main,
+                  color: theme.palette.primary.contrastText,
+                  padding: '10px 0',
+                }}
+              >
+                <Grid item xs={2.5}>
+                  <Typography align="center">Date Lost</Typography>
+                </Grid>
+                <Grid item xs={2.5}>
+                  <Typography align="center">Location</Typography>
+                </Grid>
+                <Grid item xs={2.5}>
+                  <Typography align="center">Category</Typography>
+                </Grid>
+                <Grid item xs={2.5}>
+                  <Typography align="center">Description</Typography>
+                </Grid>
+                <Grid item xs={1}>
+                  <Typography align="center">Actions</Typography>
+                </Grid>
+              </Grid>
+
+              {/* Data Rows */}
+              <Grid container spacing={2} marginTop={2}>
+                {activeReports.map((report) => (
+                  <Grid item xs={12} key={report.id}>
+                    <Card
+                      sx={{
+                        backgroundColor: theme.palette.neutral.light, // Set a proper light background
+                        marginBottom: '10px',
+                      }}
+                    >
+                      <CardContent>
+                        <Grid container spacing={2} alignItems="center">
+                          <Grid item xs={2.5}>
+                            <Typography align="center">{report.dateLost}</Typography>
+                          </Grid>
+                          <Grid item xs={2.5}>
+                            <Typography align="center">{report.location}</Typography>
+                          </Grid>
+                          <Grid item xs={2.5}>
+                            <Typography align="center">{report.category}</Typography>
+                          </Grid>
+                          <Grid item xs={2.5}>
+                            <Typography align="center">{report.description}</Typography>
```

Comment:
Might be nice to truncate the description text so that the report boxes can all be the same size. Could be something like:
{report.description.substring(0, 20)} {report.description.length >= 20 && '...'}